app/screens/modal/NoInternetConnection.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '../../basicComponents';
+import { ExternalLinks } from '../../../shared/constants';
+import Modal from '../../components/common/Modal';
+import ReactPortal from './ReactPortal';
+
+const ButtonsWrapper = styled.div<{ hasSingleButton?: boolean }>`
+  display: flex;
+  flex-direction: row;
+  justify-content: ${({ hasSingleButton }) =>
+    hasSingleButton ? 'center' : 'space-between'};
+  margin: auto 0 15px 0;
+  padding-top: 30px;
+`;
+
+const ErrorMessage = styled.pre`
+  font-size: 14px;
+  line-height: 16px;
+  word-wrap: break-word;
+  white-space: pre-wrap;
+  flex: 1;
+  overflow-y: auto;
+`;
+const getOnLineStatus = () =>
+  typeof navigator !== 'undefined' && typeof navigator.onLine === 'boolean'
+    ? navigator.onLine
+    : true;
+
+const useNavigatorOnLine = () => {
+  const [status, setStatus] = React.useState(getOnLineStatus());
+
+  React.useEffect(() => {
+    const setOnline = () => setStatus(true);
+    const setOffline = () => setStatus(false);
+    window.removeEventListener('online', setOnline);
+    window.removeEventListener('offline', setOffline);
+
+    window.addEventListener('online', setOnline);
+    window.addEventListener('offline', setOffline);
+  }, [setStatus]);
+
+  return status;
+};
+
+const NoInternetConnection = () => {
+  const navigateToDiscord = () => window.open(ExternalLinks.Discord);
+  const [isIgnore, setIgnore] = useState(false);
+
+  const isInternetConnection = useNavigatorOnLine();
+
+  if (isInternetConnection || isIgnore) {
+    return null;
+  }
+
+  return (
+    <ReactPortal modalId="no-internet-connection">
+      <Modal
+        header="WELCOME TO SPACEMESH"
+        subHeader="We cannot get the networks and connect you!"
+        width={600}
+        height={350}
+      >
+        <ErrorMessage>
+          1. Problem with internet connection on your PC.
+          {'\n'}
+          2. We are stopped all our Spacemesh networks and. We are working on
+          the connection, please wait for the new network. You can receive a
+          message about the work carried out through Discord channel
+        </ErrorMessage>
+        <ButtonsWrapper>
+          <Button onClick={navigateToDiscord} text="OPEN DISCORD CHANNEL" />
+          <Button onClick={() => setIgnore(true)} text="IGNORE" />
+        </ButtonsWrapper>
```

Comment:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/1897530/216395200-5f8b1730-3dd2-4ce0-81da-533e05fa9ac4.png">

First of all, this popup showed only when no internet connection.
So the text here is misleading. Let's tweak it a bit:

```suggestion
      <Modal
        header="NO INTERNET CONNECTION"
        width={600}
        height={350}
      >
        <ErrorMessage>
          Please, check your internet connection. Once you'll get connected this dialog should disappear automatically.
          {'\n'}
          However, you can unlock the wallet even without an internet connection. Then click the "Ignore" button and unlock the wallet as usual.
        </ErrorMessage>
        <ButtonsWrapper>
          <Button onClick={navigateToDiscord} text="OPEN DISCORD CHANNEL" width={180} />
          <Button onClick={() => setIgnore(true)} text="IGNORE" />
        </ButtonsWrapper>
```